src/main/resources/templates/metadata-management.html

```diff
@@ -53,17 +53,27 @@ <h4>Metadata Record Form</h4>
         options = /*[[${request}]]*/ {};
         /*]]>*/
 
+        var currentUrl = window.location.href;
+        var schemaId = currentUrl.substring(currentUrl.indexOf("/metadata-management/") + "/metadata-management/".length, currentUrl.length);
```

Comment:
Using the same string twice is error prone when you can use a variable instead. You should change the path anyway (see comment above)